examples/serverless-with-firebase/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "serverless-with-firebase",
+  "version": "1.0.0",
+  "description": "Next.js 8 serverless with firebase example",
+  "license": "MIT",
+  "engines": {
+    "node": "8"
+  },
+  "scripts": {
+    "dev": "next src/app",
+    "local": "run-s build local:serve",
+    "build": "run-s build:*",
+    "build:public": "cpx \"src/app/static/**/*.*\" dist/public/static -C",
+    "build:functions": "cpx \"src/functions/**/*.*\" dist/functions -C",
+    "build:app": "next build src/app",
+    "build:install": "cpx \"*{package.json,yarn.lock}\" dist/functions -C && cd dist/functions && yarn install --production",
```

Comment:
I found that you can omit the install step here.

Reply:
I'm not sure what you mean, when you deploy `dist/functions` folder to firebase cloud functions, it needs package.json file, and possible node_modules as well (not sure if firebase will install them during deployment). This is because firebase deploy only care about what's inside `dist/functions` as specified in firebase.json

Reply:
(Forgot to @johnapost )

Reply:
@shadowwalker I have a working setup now that only does the package.json and lockfile copy (still necessary). I removed my npm install command and was pleasantly surprised that it worked fine. See if it works for you.

Reply:
@johnapost Awesome, I will try your suggestion after I get some sleep 🛌.

Reply:
@johnapost Confirmed! `yarn install` is not needed in `dist/functions` folder. I have made the update, thanks for suggestion.